test/controllers/area.controller.test.js

```diff
@@ -0,0 +1,145 @@
+import { expect, request, useDatabase } from '../config';
+import AreaFactory from '../factories/area.factory';
+import UserFactory from '../factories/user.factory';
+import { addUser } from '../../server/services/user.service';
+import { USER_ROLE } from '../../server/helpers/constants';
+
+useDatabase();
+
+let userToken;
+let adminToken;
+let editorToken;
+
+const user = UserFactory.build({ role: USER_ROLE.USER, activated: true });
+const admin = UserFactory.build({ role: USER_ROLE.ADMIN, activated: true });
+const editor = UserFactory.build({ role: USER_ROLE.EDITOR, activated: true });
+
+describe('Area Controller', () => {
+  beforeEach(async () => {
+    userToken = await addUser(user);
+    adminToken = await addUser(admin);
+    editorToken = await addUser(editor);
+  });
+
+  describe('Add Area Route', () => {
+    context('when editor token is used', () => {
+      it('returns successful area', (done) => {
+        const area = AreaFactory.build();
+        request()
+          .post('/api/v1/area/add')
+          .set('authorization', editorToken)
+          .send(area)
+          .end((err, res) => {
+            expect(res).to.have.status(201);
+            expect(res.body.success).to.be.eql(true);
+            expect(res.body.message).to.be.eql('Area added successfully');
+            expect(res.body).to.have.property('area');
```

Comment:
I believe we should be able to test for the value of area returned instead of testing that the property of area exists. You can use deep equality to test for all the returned objects at once. 

You can read more about this here or google search for a better article:
https://medium.com/building-ibotta/testing-arrays-and-objects-with-chai-js-4b372310fe6d)

```suggestion
	            expect(res.body.area).to.include(area):
```